integration_tests/src/main/python/arithmetic_ops_test.py

```diff
@@ -1453,3 +1453,79 @@ def test_func(spark):
     # not we want for this test.
     conf_no_ansi = {"spark.sql.ansi.enabled": "false"}
     assert_gpu_and_cpu_are_equal_collect(test_func, conf = conf_no_ansi)
+
+
+# Test that try_* functions falls back to CPU when TRY mode is used
+@allow_non_gpu('ProjectExec')
+@pytest.mark.parametrize('data_gen', integral_gens, ids=idfn)
+def test_try_add_fallback_to_cpu(data_gen):
+    assert_gpu_fallback_collect(
+        lambda spark: binary_op_df(spark, data_gen).selectExpr(
+            "try_add(a, b) as result"), "Add")
+
+@allow_non_gpu('ProjectExec')
+@pytest.mark.parametrize('data_gen', numeric_gens, ids=idfn)
+def test_try_divide_fallback_to_cpu(data_gen):
+    assert_gpu_fallback_collect(
+        lambda spark: binary_op_df(spark, data_gen).selectExpr(
+            "try_divide(a, b) as result"), "Divide")
+
+@pytest.mark.skipif(is_before_spark_400(), reason="try_mod is not supported before Spark 4.0.0")
+@allow_non_gpu('ProjectExec')
+@pytest.mark.parametrize('data_gen', numeric_gens, ids=idfn)
+def test_try_mod_fallback_to_cpu(data_gen):
+    assert_gpu_fallback_collect(
+        lambda spark: binary_op_df(spark, data_gen).selectExpr(
+            "try_mod(a, b) as result"), "Remainder")
+
+@allow_non_gpu('ProjectExec')
+@pytest.mark.parametrize('data_gen', numeric_gens, ids=idfn)
+@pytest.mark.parametrize('try_method, op_name', [('try_add', 'Add'), ('try_divide', 'Divide')], ids=idfn)
```

Comment:
There are already tests for try_add and try_divivde above, is this test suite duplicate ?

Reply:
Yes, These are duplicate tests :( . I had added all the ops(add, divide, subtract, multiply, mod) earlier as per your suggestion earlier but had to make subtract, multiply, mod as their own tests because they were added in different Spark versions. Removed this.